### PR TITLE
Iptables controller rules

### PR DESCRIPTION
**Pull request by jdextercsc**
iptables rules for controller nodes.

TASK: [keystone | iptable rules for keystone] *********************************
ok: [wiley-ctlr-2] => (item={'comment': 'keytone public API', 'protocol': 'tcp', 'port': 5000}) => {"changed": false, "item": {"comment": "keytone public API", "port": 5000, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 5000 -m comment --comment \" keytone public API\" already exists."}
ok: [wiley-ctlr-3] => (item={'comment': 'keytone public API', 'protocol': 'tcp', 'port': 5000}) => {"changed": false, "item": {"comment": "keytone public API", "port": 5000, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 5000 -m comment --comment \" keytone public API\" already exists."}
ok: [wiley-ctlr-1] => (item={'comment': 'keytone public API', 'protocol': 'tcp', 'port': 5000}) => {"changed": false, "item": {"comment": "keytone public API", "port": 5000, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 5000 -m comment --comment \" keytone public API\" already exists."}
ok: [wiley-ctlr-2] => (item={'comment': 'keystone internal and admin api', 'protocol': 'tcp', 'port': 35357}) => {"changed": false, "item": {"comment": "keystone internal and admin api", "port": 35357, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 35357 -m comment --comment \" keystone internal and admin api\" already exists."}
ok: [wiley-ctlr-3] => (item={'comment': 'keystone internal and admin api', 'protocol': 'tcp', 'port': 35357}) => {"changed": false, "item": {"comment": "keystone internal and admin api", "port": 35357, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 35357 -m comment --comment \" keystone internal and admin api\" already exists."}
ok: [wiley-ctlr-1] => (item={'comment': 'keystone internal and admin api', 'protocol': 'tcp', 'port': 35357}) => {"changed": false, "item": {"comment": "keystone internal and admin api", "port": 35357, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 35357 -m comment --comment \" keystone internal and admin api\" already exists."}

TASK: [keystone | iptable rules for keystone] *********************************
skipping: [wiley-ctlr-2] => (item={'comment': 'keytone public API', 'protocol': 'tcp', 'port': 5000})
skipping: [wiley-ctlr-2] => (item={'comment': 'keystone internal and admin api', 'protocol': 'tcp', 'port': 35357})
skipping: [wiley-ctlr-3] => (item={'comment': 'keytone public API', 'protocol': 'tcp', 'port': 5000})
skipping: [wiley-ctlr-3] => (item={'comment': 'keystone internal and admin api', 'protocol': 'tcp', 'port': 35357})
skipping: [wiley-ctlr-1] => (item={'comment': 'keytone public API', 'protocol': 'tcp', 'port': 5000})
skipping: [wiley-ctlr-1] => (item={'comment': 'keystone internal and admin api', 'protocol': 'tcp', 'port': 35357})

TASK: [glance | iptable rules for glance] *************************************
ok: [wiley-ctlr-3] => (item={'comment': 'glance api', 'protocol': 'tcp', 'port': 9191}) => {"changed": false, "item": {"comment": "glance api", "port": 9191, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 9191 -m comment --comment \" glance api\" already exists."}
ok: [wiley-ctlr-2] => (item={'comment': 'glance api', 'protocol': 'tcp', 'port': 9191}) => {"changed": false, "item": {"comment": "glance api", "port": 9191, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 9191 -m comment --comment \" glance api\" already exists."}
ok: [wiley-ctlr-3] => (item={'comment': 'glance registry api', 'protocol': 'tcp', 'port': 9292}) => {"changed": false, "item": {"comment": "glance registry api", "port": 9292, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 9292 -m comment --comment \" glance registry api\" already exists."}
ok: [wiley-ctlr-1] => (item={'comment': 'glance api', 'protocol': 'tcp', 'port': 9191}) => {"changed": false, "item": {"comment": "glance api", "port": 9191, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 9191 -m comment --comment \" glance api\" already exists."}
ok: [wiley-ctlr-2] => (item={'comment': 'glance registry api', 'protocol': 'tcp', 'port': 9292}) => {"changed": false, "item": {"comment": "glance registry api", "port": 9292, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 9292 -m comment --comment \" glance registry api\" already exists."}
ok: [wiley-ctlr-1] => (item={'comment': 'glance registry api', 'protocol': 'tcp', 'port': 9292}) => {"changed": false, "item": {"comment": "glance registry api", "port": 9292, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 9292 -m comment --comment \" glance registry api\" already exists."}

TASK: [glance | iptable rules for glance] *************************************
skipping: [wiley-ctlr-1] => (item={'comment': 'glance api', 'protocol': 'tcp', 'port': 9191})
skipping: [wiley-ctlr-1] => (item={'comment': 'glance registry api', 'protocol': 'tcp', 'port': 9292})
skipping: [wiley-ctlr-3] => (item={'comment': 'glance api', 'protocol': 'tcp', 'port': 9191})
skipping: [wiley-ctlr-2] => (item={'comment': 'glance api', 'protocol': 'tcp', 'port': 9191})
skipping: [wiley-ctlr-3] => (item={'comment': 'glance registry api', 'protocol': 'tcp', 'port': 9292})
skipping: [wiley-ctlr-2] => (item={'comment': 'glance registry api', 'protocol': 'tcp', 'port': 9292})

TASK: [cinder | iptable rules for cinder] *************************************
ok: [wiley-ctlr-2] => (item={'comment': 'cinder api 8776', 'protocol': 'tcp', 'port': 8776}) => {"changed": false, "item": {"comment": "cinder api 8776", "port": 8776, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 8776 -m comment --comment \" cinder api 8776\" already exists."}
ok: [wiley-ctlr-3] => (item={'comment': 'cinder api 8776', 'protocol': 'tcp', 'port': 8776}) => {"changed": false, "item": {"comment": "cinder api 8776", "port": 8776, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 8776 -m comment --comment \" cinder api 8776\" already exists."}
ok: [wiley-ctlr-1] => (item={'comment': 'cinder api 8776', 'protocol': 'tcp', 'port': 8776}) => {"changed": false, "item": {"comment": "cinder api 8776", "port": 8776, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 8776 -m comment --comment \" cinder api 8776\" already exists."}
ok: [wiley-ctlr-2] => (item={'comment': 'iscsi traffic incoming 3260', 'protocol': 'tcp', 'port': 3260}) => {"changed": false, "item": {"comment": "iscsi traffic incoming 3260", "port": 3260, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 3260 -m comment --comment \" iscsi traffic incoming 3260\" already exists."}
ok: [wiley-ctlr-3] => (item={'comment': 'iscsi traffic incoming 3260', 'protocol': 'tcp', 'port': 3260}) => {"changed": false, "item": {"comment": "iscsi traffic incoming 3260", "port": 3260, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 3260 -m comment --comment \" iscsi traffic incoming 3260\" already exists."}
ok: [wiley-ctlr-1] => (item={'comment': 'iscsi traffic incoming 3260', 'protocol': 'tcp', 'port': 3260}) => {"changed": false, "item": {"comment": "iscsi traffic incoming 3260", "port": 3260, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 3260 -m comment --comment \" iscsi traffic incoming 3260\" already exists."}

TASK: [neutron | iptable rules for neutron] ***********************************
ok: [wiley-ctlr-2] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 67}) => {"changed": false, "item": {"comment": "dhcp incoming", "port": 67, "protocol": "udp"}, "msg": "rule INPUT -p udp --dport 67 -m comment --comment \" dhcp incoming\" already exists."}
ok: [wiley-ctlr-3] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 67}) => {"changed": false, "item": {"comment": "dhcp incoming", "port": 67, "protocol": "udp"}, "msg": "rule INPUT -p udp --dport 67 -m comment --comment \" dhcp incoming\" already exists."}
ok: [wiley-ctlr-1] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 67}) => {"changed": false, "item": {"comment": "dhcp incoming", "port": 67, "protocol": "udp"}, "msg": "rule INPUT -p udp --dport 67 -m comment --comment \" dhcp incoming\" already exists."}
ok: [wiley-ctlr-2] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 688888888}) => {"changed": false, "item": {"comment": "dhcp incoming", "port": 688888888, "protocol": "udp"}, "msg": "rule INPUT -p udp --dport 688888888 -m comment --comment \" dhcp incoming\" already exists."}
ok: [wiley-ctlr-3] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 688888888}) => {"changed": false, "item": {"comment": "dhcp incoming", "port": 688888888, "protocol": "udp"}, "msg": "rule INPUT -p udp --dport 688888888 -m comment --comment \" dhcp incoming\" already exists."}
ok: [wiley-ctlr-3] => (item={'comment': 'neutron api', 'protocol': 'tcp', 'port': 9696}) => {"changed": false, "item": {"comment": "neutron api", "port": 9696, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 9696 -m comment --comment \" neutron api\" already exists."}
ok: [wiley-ctlr-2] => (item={'comment': 'neutron api', 'protocol': 'tcp', 'port': 9696}) => {"changed": false, "item": {"comment": "neutron api", "port": 9696, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 9696 -m comment --comment \" neutron api\" already exists."}
ok: [wiley-ctlr-1] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 688888888}) => {"changed": false, "item": {"comment": "dhcp incoming", "port": 688888888, "protocol": "udp"}, "msg": "rule INPUT -p udp --dport 688888888 -m comment --comment \" dhcp incoming\" already exists."}
ok: [wiley-ctlr-3] => (item={'comment': 'neutron tunnels', 'protocol': 'tcp', 'port': 4789}) => {"changed": false, "item": {"comment": "neutron tunnels", "port": 4789, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 4789 -m comment --comment \" neutron tunnels\" already exists."}
ok: [wiley-ctlr-2] => (item={'comment': 'neutron tunnels', 'protocol': 'tcp', 'port': 4789}) => {"changed": false, "item": {"comment": "neutron tunnels", "port": 4789, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 4789 -m comment --comment \" neutron tunnels\" already exists."}
ok: [wiley-ctlr-1] => (item={'comment': 'neutron api', 'protocol': 'tcp', 'port': 9696}) => {"changed": false, "item": {"comment": "neutron api", "port": 9696, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 9696 -m comment --comment \" neutron api\" already exists."}
ok: [wiley-ctlr-1] => (item={'comment': 'neutron tunnels', 'protocol': 'tcp', 'port': 4789}) => {"changed": false, "item": {"comment": "neutron tunnels", "port": 4789, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 4789 -m comment --comment \" neutron tunnels\" already exists."}

TASK: [neutron | iptable rules for neutron] ***********************************
skipping: [wiley-ctlr-2] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 67})
skipping: [wiley-ctlr-2] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 68})
skipping: [wiley-ctlr-2] => (item={'comment': 'neutron api', 'protocol': 'tcp', 'port': 9696})
skipping: [wiley-ctlr-2] => (item={'comment': 'neutron tunnels', 'protocol': 'tcp', 'port': 4789})
skipping: [wiley-ctlr-1] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 67})
skipping: [wiley-ctlr-3] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 67})
skipping: [wiley-ctlr-1] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 68})
skipping: [wiley-ctlr-3] => (item={'comment': 'dhcp incoming', 'protocol': 'udp', 'port': 68})
skipping: [wiley-ctlr-1] => (item={'comment': 'neutron api', 'protocol': 'tcp', 'port': 9696})
skipping: [wiley-ctlr-3] => (item={'comment': 'neutron api', 'protocol': 'tcp', 'port': 9696})
skipping: [wiley-ctlr-1] => (item={'comment': 'neutron tunnels', 'protocol': 'tcp', 'port': 4789})
skipping: [wiley-ctlr-3] => (item={'comment': 'neutron tunnels', 'protocol': 'tcp', 'port': 4789})

TASK: [nova-control | iptable rules for nova-controller] **********************
ok: [wiley-ctlr-2] => (item={'comment': 'novncproxy', 'protocol': 'tcp', 'port': 6080}) => {"changed": false, "item": {"comment": "novncproxy", "port": 6080, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 6080 -m comment --comment \" novncproxy\" already exists."}
ok: [wiley-ctlr-3] => (item={'comment': 'novncproxy', 'protocol': 'tcp', 'port': 6080}) => {"changed": false, "item": {"comment": "novncproxy", "port": 6080, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 6080 -m comment --comment \" novncproxy\" already exists."}
ok: [wiley-ctlr-1] => (item={'comment': 'novncproxy', 'protocol': 'tcp', 'port': 6080}) => {"changed": false, "item": {"comment": "novncproxy", "port": 6080, "protocol": "tcp"}, "msg": "rule INPUT -p tcp --dport 6080 -m comment --comment \" novncproxy\" already exists."}
changed: [wiley-ctlr-2] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': 8774}) => {"changed": true, "item": {"comment": "nova api", "port": 8774, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8774 -m comment --comment \" nova api\""}
changed: [wiley-ctlr-3] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': 8774}) => {"changed": true, "item": {"comment": "nova api", "port": 8774, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8774 -m comment --comment \" nova api\""}
changed: [wiley-ctlr-2] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': 8775}) => {"changed": true, "item": {"comment": "nova api", "port": 8775, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8775 -m comment --comment \" nova api\""}
changed: [wiley-ctlr-3] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': 8775}) => {"changed": true, "item": {"comment": "nova api", "port": 8775, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8775 -m comment --comment \" nova api\""}
changed: [wiley-ctlr-1] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': 8774}) => {"changed": true, "item": {"comment": "nova api", "port": 8774, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8774 -m comment --comment \" nova api\""}
changed: [wiley-ctlr-3] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': 8773}) => {"changed": true, "item": {"comment": "nova api", "port": 8773, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8773 -m comment --comment \" nova api\""}
changed: [wiley-ctlr-2] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': 8773}) => {"changed": true, "item": {"comment": "nova api", "port": 8773, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8773 -m comment --comment \" nova api\""}
changed: [wiley-ctlr-1] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': 8775}) => {"changed": true, "item": {"comment": "nova api", "port": 8775, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8775 -m comment --comment \" nova api\""}
changed: [wiley-ctlr-1] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': 8773}) => {"changed": true, "item": {"comment": "nova api", "port": 8773, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8773 -m comment --comment \" nova api\""}

TASK: [nova-control | iptable rules for nova-controller] **********************
skipping: [wiley-ctlr-1] => (item={'comment': 'novncproxy', 'protocol': 'tcp', 'port': '6080'})
skipping: [wiley-ctlr-2] => (item={'comment': 'novncproxy', 'protocol': 'tcp', 'port': '6080'})
skipping: [wiley-ctlr-1] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': '8773,8774,8775'})
skipping: [wiley-ctlr-3] => (item={'comment': 'novncproxy', 'protocol': 'tcp', 'port': '6080'})
skipping: [wiley-ctlr-2] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': '8773,8774,8775'})
skipping: [wiley-ctlr-3] => (item={'comment': 'nova api', 'protocol': 'tcp', 'port': '8773,8774,8775'})

TASK: [heat | iptable rules for heat] *****************************************
changed: [wiley-ctlr-3] => (item={'comment': 'cloudwatch API', 'protocol': 'tcp', 'port': 8003}) => {"changed": true, "item": {"comment": "cloudwatch API", "port": 8003, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8003 -m comment --comment \" cloudwatch API\""}
changed: [wiley-ctlr-2] => (item={'comment': 'cloudwatch API', 'protocol': 'tcp', 'port': 8003}) => {"changed": true, "item": {"comment": "cloudwatch API", "port": 8003, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8003 -m comment --comment \" cloudwatch API\""}
changed: [wiley-ctlr-1] => (item={'comment': 'cloudwatch API', 'protocol': 'tcp', 'port': 8003}) => {"changed": true, "item": {"comment": "cloudwatch API", "port": 8003, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8003 -m comment --comment \" cloudwatch API\""}
changed: [wiley-ctlr-3] => (item={'comment': 'heat api', 'protocol': 'tcp', 'port': 8004}) => {"changed": true, "item": {"comment": "heat api", "port": 8004, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8004 -m comment --comment \" heat api\""}
changed: [wiley-ctlr-2] => (item={'comment': 'heat api', 'protocol': 'tcp', 'port': 8004}) => {"changed": true, "item": {"comment": "heat api", "port": 8004, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8004 -m comment --comment \" heat api\""}
changed: [wiley-ctlr-3] => (item={'comment': 'cfn api', 'protocol': 'tcp', 'port': 8000}) => {"changed": true, "item": {"comment": "cfn api", "port": 8000, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8000 -m comment --comment \" cfn api\""}
changed: [wiley-ctlr-2] => (item={'comment': 'cfn api', 'protocol': 'tcp', 'port': 8000}) => {"changed": true, "item": {"comment": "cfn api", "port": 8000, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8000 -m comment --comment \" cfn api\""}
changed: [wiley-ctlr-1] => (item={'comment': 'heat api', 'protocol': 'tcp', 'port': 8004}) => {"changed": true, "item": {"comment": "heat api", "port": 8004, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8004 -m comment --comment \" heat api\""}
changed: [wiley-ctlr-1] => (item={'comment': 'cfn api', 'protocol': 'tcp', 'port': 8000}) => {"changed": true, "item": {"comment": "cfn api", "port": 8000, "protocol": "tcp"}, "msg": "INPUT -p tcp --dport 8000 -m comment --comment \" cfn api\""}

TASK: [heat | iptable rules for heat] *****************************************
skipping: [wiley-ctlr-1] => (item={'comment': 'cloudwatch API', 'protocol': 'tcp', 'port': 8003})
skipping: [wiley-ctlr-1] => (item={'comment': 'heat api', 'protocol': 'tcp', 'port': 8004})
skipping: [wiley-ctlr-1] => (item={'comment': 'cfn api', 'protocol': 'tcp', 'port': 8000})
skipping: [wiley-ctlr-2] => (item={'comment': 'cloudwatch API', 'protocol': 'tcp', 'port': 8003})
skipping: [wiley-ctlr-3] => (item={'comment': 'cloudwatch API', 'protocol': 'tcp', 'port': 8003})
skipping: [wiley-ctlr-2] => (item={'comment': 'heat api', 'protocol': 'tcp', 'port': 8004})
skipping: [wiley-ctlr-3] => (item={'comment': 'heat api', 'protocol': 'tcp', 'port': 8004})
skipping: [wiley-ctlr-2] => (item={'comment': 'cfn api', 'protocol': 'tcp', 'port': 8000})
skipping: [wiley-ctlr-3] => (item={'comment': 'cfn api', 'protocol': 'tcp', 'port': 8000})

TASK: [ceilometer-control | iptable rules for ceilometer-controller] **********
changed: [wiley-ctlr-2] => {"changed": true, "msg": "INPUT -p tcp --dport 8777 -m comment --comment \" Ceilometer incoming\""}
changed: [wiley-ctlr-3] => {"changed": true, "msg": "INPUT -p tcp --dport 8777 -m comment --comment \" Ceilometer incoming\""}
changed: [wiley-ctlr-1] => {"changed": true, "msg": "INPUT -p tcp --dport 8777 -m comment --comment \" Ceilometer incoming\""}

TASK: [horizon | iptable rules for ceilometer-controller] *********************
changed: [wiley-ctlr-2] => {"changed": true, "msg": "INPUT -p tcp --dport 80 -m comment --comment \" Dashboard access\""}
changed: [wiley-ctlr-3] => {"changed": true, "msg": "INPUT -p tcp --dport 80 -m comment --comment \" Dashboard access\""}
changed: [wiley-ctlr-1] => {"changed": true, "msg": "INPUT -p tcp --dport 80 -m comment --comment \" Dashboard access\""}

TASK: [horizon | iptable rules for ceilometer-controller] *********************
skipping: [wiley-ctlr-3]
skipping: [wiley-ctlr-2]
skipping: [wiley-ctlr-1]

PLAY RECAP ********************************************************************
wiley-ctlr-1               : ok=14   changed=4    unreachable=0    failed=0
wiley-ctlr-2               : ok=14   changed=4    unreachable=0    failed=0
wiley-ctlr-3               : ok=14   changed=4    unreachable=0    failed=0

[root@wiley-ctlr-1 ~]# iptables -nL
Chain INPUT (policy ACCEPT)
target     prot opt source               destination
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:80 /\*  Dashboard access _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:8777 /_  Ceilometer incoming _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:8000 /_  cfn api _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:8004 /_  heat api _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:8003 /_  cloudwatch API _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:8773 /_  nova api _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:8775 /_  nova api _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:8774 /_  nova api _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:6080 /_  novncproxy _/
ACCEPT     udp  --  0.0.0.0/0            0.0.0.0/0            udp dpt:39992 /_  dhcp incoming _/
ACCEPT     udp  --  0.0.0.0/0            0.0.0.0/0            udp dpt:67 /_  dhcp incoming _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:4789 /_  neutron tunnels _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:9696 /_  neutron api _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:3260 /_  iscsi traffic incoming 3260 _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:8776 /_  cinder api 8776 _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:9292 /_  glance registry api _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:9191 /_  glance api _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:35357 /_  keystone internal and admin api _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:5000 /_  keytone public API */
nova-api-INPUT  all  --  0.0.0.0/0            0.0.0.0/0

---

ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:6379 /\*  redis replication for mongodb _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:27017 /_  incoming mongodb _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:4369 /_  epmd cluster _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:25672 /_  epmd cluster _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:5672 /_  amqp traffic / rabbitmq _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:11211 /_  memcache incoming _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:5405 /_  Pacemaker ports _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:21064 /_  Pacemaker ports _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:3121 /_  Pacemaker ports _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:2224 /_  Pacemaker ports _/
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:22 /_  ssh incoming */

### CODE REVIEW

File: roles/neutron/tasks/juno.yml [hunk 1]
```diff
@@ -1,4 +1,18 @@
 ---
+- name: iptable rules for neutron
+  iptables:
+    port: "{{ item.port }}"
+    protocol: "{{ item.protocol}}"
+    comment: "{{item.comment}}"
+  with_items:
+    - { port: 67, protocol: udp, comment: "dhcp incoming" }
+    - { port: 688888888, protocol: udp, comment: "dhcp incoming" }
```
Comment (jcpowermac) [inline roles/neutron/tasks/juno.yml]: @jdextercsc is that an oops? 688888888

Comment (jdextercsc) [inline roles/neutron/tasks/juno.yml]: That would be a vim macro looks like....

**Comment by jcpowermac** [comment]
Code Review :+1:

**Comment by mtnbikenc** [comment]
Code review :+1:

**Pull request merged**